client/src/Components/LanguageSelectMenu.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import InputLabel from '@material-ui/core/InputLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormControl from '@material-ui/core/FormControl';
+import Select from '@material-ui/core/Select';
+import { useStyles } from '../themes/theme';
+
+const LanguageSelectMenu = (props) => {
+  const classes = useStyles()
+
+  return (
+    <FormControl className={classes[`form-dropdown`]} variant="outlined">
+      <InputLabel id="demo-simple-select-outlined-label">Language</InputLabel>
+      <Select
+        labelId="demo-simple-select-outlined-label"
+        id="demo-simple-select-outlined"
+        value={props.language}
+        onChange={props.handleLanguageChange}
+        label="Language"
+      >
+        <MenuItem value={'javascript'}>Javascript</MenuItem>
+        <MenuItem value={'python'}>Python</MenuItem>
```

Comment:
This should come from a list in a utils folder or something so that we can easily change it and also keep it in one place to modify /review